include/libdnf/rpm/reldep.hpp

```diff
@@ -29,7 +29,19 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 namespace libdnf::rpm {
 
-class ReldepListIterator;
+class PackageSack;
+using PackageSackWeakPtr = WeakPtr<PackageSack, false>;
+
+struct ReldepId {
```

Comment:
I have the same comment as for PackageId. Not sure whether it is the best place.